lib/login/accountCreation.dart

```diff
@@ -0,0 +1,216 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import 'package:http/http.dart' as http;
+import 'package:off_top_mobile/login/login.dart';
+
+int repoNumber;
```

Comment:
It seems that this isn't used at all in accountCreation.dart, will anything break if you remove it?

Reply:
@tonynguyen98 If you remove the imports, you will not be able to make post request. As for repoNumber it  will be removed.

Reply:
Apologies there, accidently selected more than one lines, I was just referencing line 9